examples/sidecar/.happy/terraform/envs/rdev/providers.tf

```diff
@@ -0,0 +1,44 @@
+# This file is autogenerated by 'happy infra generate'. Do not edit manually.
```

Comment:
We need to update this comment so that Github picks it up as a generated file. See the fogg comments to match. They are ignored by Github already

Reply:
```
# Auto-generated by 'happy infra'. Do not edit
# Make improvements in happy, so that everyone can benefit.
```
?

Reply:
I tried to find docs on how Github knows a file is autogenerate but didn't see anything. I think this is fine